### PR TITLE
MM-20977 - Inviting multiple users with valid/allowed and invalid emails causes the invites for the valid users not to be sent

### PR DESCRIPTION
**Pull request by reflog**
#### Summary
Added addUsersToTeamGracefully to allow mixing bad/good users in invite

#### Ticket Link
Fixes https://mattermost.atlassian.net/browse/MM-20977

#### Related PRs
Server: https://github.com/mattermost/mattermost-server/pull/13372
Webapp: https://github.com/mattermost/mattermost-webapp/pull/4495

### CODE REVIEW

File: src/actions/teams.ts [hunk 1]
```diff
@@ -461,6 +461,35 @@ export function addUsersToTeam(teamId: string, userIds: Array<string>): ActionFu
...
+
+        dispatch(batchActions([
+            {
+                type: UserTypes.RECEIVED_PROFILES_LIST_IN_TEAM,
+                data: profiles,
+                id: teamId,
+            },
+            {
+                type: TeamTypes.RECEIVED_MEMBERS_IN_TEAM,
+                data: result,
```
Comment (jespino) [inline src/actions/teams.ts]: Why not using here something like `result.added_members || []`?

**Comment by hanzei** [comment]
@reflog Did this got cherry picked? Should this get cherry picked?

**Comment by reflog** [comment]
@hanzei - it was cherry picked to 5.18. It's not breaking anything there, just adds a (currently unused) function

**Comment by hanzei** [comment]
Thanks for confirming. Just wanted to double check. :+1:

**Pull request merged**